src/Cohesity.Powershell/Scripts/VirtualIP/New-CohesityVirtualIP.ps1

```diff
@@ -0,0 +1,78 @@
+function New-CohesityVirtualIP {
+    <#
+        .SYNOPSIS
+        Creates new virtual IP(s).
+        .DESCRIPTION
+        The New-CohesityVirtualIP function is used to create virtual IP(s).
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        New-CohesityVirtualIP -VlanName "intf_group2.0" -Subnet "10.3.144.0" -NetmaskBitsForSubnet 20 -Gateway "10.3.144.1" -VirtualIPs "10.3.144.14", "10.3.144.15"
+    #>
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory = $true)]
```

Comment:
Are all the parameters mandatory ?

Reply:
Fixed: Necessary parameters are made mandatory